src/en/contributing/resources.md

```diff
@@ -1,23 +1,3 @@
-# Goals
-
-- Come up with a name / codename.
-- Must have communities.
-- Must have threaded comments.
-- Must be federated: liking and following communities across instances.
-- Be live-updating: have a right pane for new comments, and a main pane for the full threaded view.
-  - Use websockets for post / gets to your own instance.
-
-# Questions
-
-- How does voting work? Should we go back to the old way of showing up and downvote counts? Or just a score?
-- Decide on tech to be used
-  - Backend: Actix, Diesel.
-  - Frontend: inferno, typescript and bootstrap for now.
-- Should it allow bots?
-- Should the comments / votes be static, or feel like a chat, like [flowchat?](https://flow-chat.com).
-  - Two pane model - Right pane is live comments, left pane is live tree view.
-  - On mobile, allow you to switch between them. Default?
-
 # Resources / Potential Libraries
```

Comment:
If you want to keep the resources list, it could use some updates. Like including links about activitypub. And just remove the part "potential libraries" from the title.

Reply:
The activitypub has its own resources in the federation folder. I can update the title tho.

Reply:
Right, then lets merge them? Categorizing them would also be nice.

Reply:
I could move those over to this one I spose.